types/src/chain_id.rs

```diff
@@ -86,10 +96,35 @@ where
     deserializer.deserialize_any(ChainIdVisitor)
 }
 
+impl Debug for ChainId {
+    fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
+        write!(f, "{}", self)
+    }
+}
+
 impl Display for ChainId {
     fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
-        // TODO add pretty printing for NamedChain
-        write!(f, "ChainId {:?}", self.0)
+        write!(
+            f,
+            "{}",
+            NamedChain::from_chain_id(&self)
+                .map_or_else(|_| self.0.to_string(), |chain| chain.to_string())
+        )
+    }
+}
+
+impl Display for NamedChain {
+    fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
+        write!(
+            f,
+            "{}",
+            match self {
+                NamedChain::DEVNET => "Devnet",
+                NamedChain::TESTNET => "Testnet",
+                NamedChain::MAINNET => "Mainnet",
+                NamedChain::TESTING => "Testing",
```

Comment:
Can you humor me and test out the config-builder faucet command with a chain-id Testnet versus TESTNET, because we're using TESTNET internally.

Reply:
Ah, yes this is problematic on my side, it's all caps in the other direction.  will fix